packages/angular/stories/generated/forms.mdx

```diff
@@ -0,0 +1,595 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Components/Forms" />
+
+# Components/Forms
+
+{/*
+This file is automatically generated from forms.stories.ts
+Do not edit this file directly. Run 'yarn extract-templates' to regenerate.
+To add custom introduction content, create forms.intro.md */}
+
+## All Reactive
+
+An example form that includes all form controls using reactive forms.
+
+
+```typescript
```

Comment:
Maybe it would be helpful to show people here from where they should import components, directives?

sth like
```
import { CheckboxDirective, CheckboxGroupDirective, RadioGroupDirective, SelectDirective, TextFieldDirective } from '@sl-design-system/angular/forms';
import { TextFieldComponent } from '@sl-design-system/angular/text-field';
import { SelectComponent } from '@sl-design-system/angular/select';
...
```